app/src/main/java/no/usn/mob3000/data/repository/social/ProfileEditRepository.kt

```diff
@@ -0,0 +1,60 @@
+package no.usn.mob3000.data.repository.social
+
+import kotlinx.datetime.Clock
+import no.usn.mob3000.data.model.social.ProfileDto
+import no.usn.mob3000.data.source.remote.auth.UserDataSource
+import no.usn.mob3000.data.source.remote.social.ProfileUserDataSource
+import no.usn.mob3000.domain.repository.social.IProfileEditRepository
+/**
+ *
+ * This repository orchestrates the updating of the users profile. Currently there is only one implementation of
+ * updating a table under profile, thus [updateProfile] being the sole function here
+ *
+ * @param profileEditDataSource The data source for fetching friends.
+ * @author 258030
+ * @created 2024-11-15
+ */
+class ProfileEditRepository (
+    private val profileEditDataSource: ProfileUserDataSource = ProfileUserDataSource(),
+    private val userDataSource: UserDataSource = UserDataSource()
+): IProfileEditRepository{
```

Comment:
This is not a return data type, it's a class implementing/extending something. This is incorrect. Starting bracket for the class is also incorrectly spaced.

Reply:
Are you talking about KDoc or the actual functionality of the class? Can't see the same as you do

Reply:
It is literally the line I'm commenting on. The placement of something being implemented or extended, versus being an expected result is well defined. A class implementing `IProfileEditRepository` does not expect it as a result, hence the placement is wrong. It should be `) : IPr`. Additionally, the starting bracket for the class is incorrectly placed. It should be visibly separated from the implementation by a space. It should be `ory {`.